app/views/partials/_header.html.erb

```diff
@@ -13,8 +13,7 @@
       <% if session[:favorites].nil? %>
         <%= link_to "favorites: 0", "/favorites", class: "navbar-item" %>
       <% else %>
-        <p class= "navbar-item">favorites: <%= favorites.count %></p>
-        <%= link_to "favorites: #{session[:favorites].size}","/favorites" ,class: "navbar-item"%>
+        <%= link_to "favorites: #{favorites.count}","/favorites" ,class: "navbar-item"%>
```

Comment:
Nice catches Kwibe! Going to the website to fix errors in the front end seems like the way to do it for sure!